index.js

```diff
@@ -10,5 +10,6 @@ app.get("/ping", (req, res) => {
     });
 });
 
-app.use("/users", require("./routes/users_routes"));
+app.use("/users", require("./src/routes/users_routes"));
+module.exports = app;
```

Comment:
Can this be on the last line? I'd prefer if it was.

Reply:
sure 👍